toolkit/smart-contracts/offchain/src/d_param/mod.rs

```diff
@@ -52,19 +59,26 @@ impl<C: QueryLedgerState + QueryNetwork + Transactions + QueryUtxoByUtxoId> Upse
 	}
 }
 
+/// This function upserts D-param.
+/// Arguments:
+///  - `genesis_utxo`: UTxO identifying the Partner Chain.
+///  - `d_parameter`: [DParameter] to be upserted.
+///  - `payment_signing_key`: Signing key of a governance authority member.
+///  - `client`: Bridge client.
```

Comment:
"Bridge client" doesn't really explain anything. If not "ogmios", then I would call it "Cardano network client". As long as we don't provide any real alternative to Ogmios, I think it is easier to understand when concrete names are used.